eventdata/challenges/default.json

```diff
@@ -29,8 +29,23 @@
           "clients": {{bulk_indexing_clients | default(8)}}
         },
         {
-          "operation": "force-merge",
-          "clients": 1
```

Comment:
Nit:

Removing `"clients": 1`, even though the right thing as [it is the default](https://esrally.readthedocs.io/en/stable/track.html#schedule) makes things a bit inconsistent as elsewhere (in most of the other tasks) we still have `"clients": 1`.
Does it make sense to retain it and in a follow-up PR remove all the `"clients": 1` in tasks?

Reply:
I thought I take the opportunity to remove some unnecessary noise here. As discussed elsewhere I will create a follow-up PR which removes the rest of these lines.

Reply:
I've created https://github.com/elastic/rally-tracks/pull/105.